api/src/main/java/org/openmrs/api/PatientService.java

```diff
@@ -844,4 +845,13 @@ public List<Patient> getPatients(String name, String identifier, List<PatientIde
 	 * @throws PatientIdentifierTypeLockedException
 	 */
 	public void checkIfPatientIdentifierTypesAreLocked() throws PatientIdentifierTypeLockedException;
-}
\ No newline at end of file
+
+	/**
+	 * Get all patientIdentifiers that are associated to the patient program
+	 * @param patientProgram the patientProgram to be used to fetch the associated identifiers
```

Comment:
`@param patientProgram the patient program to be used to fetch the associated identifiers`